src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/visualeditor/VisualEditorPageObject.java

```diff
@@ -559,4 +564,16 @@ public VisualEditorAddMediaDialog searchImage(String searchInput) {
     return mediaDialog;
   }
 
+  public VisualEditorPageObject  getTextArea() {
+    infoboxNode.click();
+    return this;
+  }
+
+  public VisualEditorPageObject  getInfoboxNode() {
+    infoboxPopupCollapsed.click();
+    return this;
+  }
 }
+
+
+
```

Comment:
Unneeded newlines. Please check the code style conventions for this pull request